masquerade-sample/src/main/java/com/flipkart/masquerade/test/TestConfig.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2017 Flipkart Internet, pvt ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.masquerade.test;
+
+import com.flipkart.masquerade.Configuration;
+import com.flipkart.masquerade.rule.*;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Created by shrey.garg on 25/04/17.
+ */
+public class TestConfig implements Configuration {
```

Comment:
Would be good to complete this example by actually using the generated Cloak, passing real Eval objects and printing the result One, Two, Three objects for various Eval objects.

Also an example object having Map, List whose values or elements can be cloaked as well will demonstrate capabilities.